src/Cubes.ts

```diff
@@ -161,14 +161,14 @@ const cubeUpgradeDescriptions = [
     '[5x1] Yeah yeah yeah, +5% score on Ascension per level. Isn\'t it enough?',
     '[5x2] You now gain +4% Obtainium per level, which is not dependent on corruptions!',
     '[5x3] Gain another +3% corruption-independent Obtainium per level.',
-    '[5x4] Blah blah blah Midas works harder (same rewards as before)',
-    '[5x5] Blah blah blah Hermes works harder (same rewards as before)',
+    '[5x4] Blah blah blah Midas works harder (Same rewards as before)',
+    '[5x5] Blah blah blah Hermes works harder (Same rewards as before)',
```

Comment:
s

Reply:
Not changin, as every other simular case have Big letter inside () or []

Reply:
then it's not getting merged